examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Oh, ok, missed that. Yes and, iirc, it was in the asset fetcher proposal and then it was dropped at some point. Let me check the reason behind the drop.